### PR TITLE
[IMP] l10n_au: update fiscal year last day, label display, etc

### PR DESCRIPTION
**Pull request by tong-odoo**
task-3187734

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by vin-odoo — CHANGES_REQUESTED** [review]
I have a few comments for you :)

File: addons/l10n_au/views/res_company_views.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='vat']" position="attributes">
+                <attribute name="nolabel">1</attribute>
+            </xpath>
+            <field name="vat" position="before">
+                <label for="vat" attrs="{'invisible':[('country_code', '=', 'AU')]}" />
+                <label for="vat" string="ABN" attrs="{'invisible':[('country_code', '!=', 'AU')]}" />
+            </field>
```
[the remark this answers is not in the document]
Comment (tong-odoo) [reply addons/l10n_au/views/res_company_views.xml:15]: Sure I will update it once vat_label is deployed to 16

**Comment by tong-odoo** [comment]
Hi @vin-odoo. Thanks for the review! I have updated the PR based on the comments. Please take a look :smile:

**Review by vin-odoo — APPROVED** [review]
The tasks about labels aren't moving as fast as I would like, so let's merge this one and I'll update it later if we ever release them :smile: 

Thank you for the nice work :)

@robodoo r+

**Pull request closed without merging**